docs/fava.rst

```diff
@@ -59,36 +70,38 @@ The value of the pydata key is a dictionary of the pydata to be added to the res
 
 ### Fava and Loops ###
 
+**************
+Fava and Loops
+**************
+
 There are no loops in Fava (no for-loop, no while-loop, ...).   These kinds of loops are a major pain to implement, and are a major source of bugs for both implementers and users.  All loops that a rule writer might want to write can be written as an expression on lists (a transformation of one list to another, or a transformation of a list to a single value).  Expressions on lists are easy to implement, and easy to use once you know how.  So expressions on lists is what Fava has.
 
-Take this Python for loop for example:
+Take this Python for loop for example::
 
-```python
-up = []
-for e in eth:
-  if e.link_detected:
-    up.append(e)
-```
+  up = []
+  for e in eth:
+    if e.link_detected:
+      up.append(e)
 
-This creates a list, up, which contains every member of the list eth for which the member link_detected is true.  It transforms the list eth to the list up.
+This creates a list, up, which contains every member of the list eth for which the member link_detectedis true.  It transforms the list eth to the list up.
 
-In Fava we write this:
+In Fava we write this::
 
-```yaml
----
-rule:
-  # stuff skipped....
-  vars:
-    up: "{{ eth|selectattr('link_detected')|list }}"
-  # stuff skipped....
-```
+  ---
+  rule:
+    # stuff skipped....
+    vars:
+      up: "{{ eth|selectattr('link_detected')|list }}"
+    # stuff skipped....
 
 This is a Jinja2 expression, it says to apply the 'selectattr' filter to 'eth', and then to apply the 'list' filter to its result.  The 'selectattr' filter selects items from its argument (the thing it is filtering), including in its result only those for which the given attribute ('link_detected') is true.  The 'list' filter converts it's argument to an actual list if it isn't already.  Jinja2 will optimize a series of fiters to avoid creating intermediate lists for each step.
```

Comment:
Did you mean to emphasize terms using backticks like this `selectattr` instead of single quotes 'selectattr'?